src/lib.rs

```diff
@@ -1413,54 +1413,48 @@ impl<T: Unaligned + Display> Display for Unalign<T> {
     }
 }
 
-// Used in `transmute!` below.
 #[doc(hidden)]
-pub use core::mem::transmute as __real_transmute;
+pub trait TransmuteFrom<Src>: Sized {
```

Comment:
This trait does not need to be (and shouldn't be) `pub`, because it has a blanket implementation on `T: Sized` that `transmute` can use.